openmc/mgxs_library.py

```diff
@@ -856,9 +857,35 @@ def set_scatter_mgxs(self, scatter, nuclide='total', xs_type='macro'):
         check_value('domain_type', scatter.domain_type,
                     ['universe', 'cell', 'material'])
 
+        # Methods of representing anisotropic scattering besides
```

Comment:
I think this comment is basically redundant with the error message, they both seem to say the same thing.